lib/media/track/remotemediatrack.js

```diff
@@ -110,6 +136,7 @@ function mixinRemoteMediaTrack(AudioOrVideoTrack) {
  * @event RemoteMediaTrack#unsubscribed
  * @param {RemoteMediaTrack} track - The {@link RemoteMediaTrack} that was
  *   unsubscribed from
+ * @deprecated
```

Comment:
We should suggest listening to the event on the RemoteTrackPublication; however, I think we may want to actually open an IDL PR to add this back (or at least keep it in JavaScript). I think there are scenarios where it's more convenient to do this:

```js
participant.on('trackSubscribed', trackSubscribed);

function trackSubscribed(track) {
  document.body.appendChild(track.attach());
  track.once('unsubscribed', () => trackUnsubscribed(track));
}

function trackUnsubscribed(track) {
  track.detach().forEach(el => el.remove());
}
```

than this:

```js
participant.on('trackSubscribed', trackSubscribed);
participant.on('trackUnsubscribed', trackUnsubscribed);

function trackSubscribed(track) {
  document.body.appendChild(track.attach());
}

function trackUnsubscribed(track) {
  track.detach().forEach(el => el.remove());
}
```

Nevertheless, let's not hold up this PR. We can always add it back in a minor release if it's important.